runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2842,9 +2842,12 @@ configurateGCWithPolicyAndOptionsStandard(MM_EnvironmentBase *env)
 MM_Configuration *
 configurateGCWithPolicyAndOptions(OMR_VM* omrVM)
 {
+	J9JavaVM *vm = (J9JavaVM*) omrVM->_language_vm;
+	PORT_ACCESS_FROM_JAVAVM(vm);
 	MM_Configuration *result = NULL;
 	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(omrVM);
 	MM_EnvironmentBase env(omrVM);
+	bool ccMark;
```

Comment:
sorry, to nitpick, per coding standards, should be initialized to some value, say false

Reply:
done.